frontend/react/src/wrapSecurity.js

```diff
@@ -34,16 +36,18 @@ const WrappedSecurity = () => {
       </div>
     )
   } else {
+
     return (
       <Router>
         <Security {...config.oidc}>
```

Comment:
```suggestion
        <Security {...config.oidc} tokenManager={{ storage: "cookie" }}>
```

This will prevent Okta from storing the token in localStorage. Okta will still write other stuff there, but it's endpoint and key cacheing, so that should be okay.

Reply:
Trying this out locally now, thanks